portfolio/src/main/java/com/google/sps/data/ContactMe.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+public final class ContactMe {
+
+    private final long id;
+    private final String firstName;
+    private final String lastName;
+    private final String title;
+    private final String subject;
+    private final long timestamp;
+
+    public ContactMe(long id, String title, long timestamp, String firstname, String lastName, String subject) {
+    this.id = id;
+    this.title = title;
+    this.timestamp = timestamp;
+    this.firstName = firstname;
+    this.lastName = lastName;
+    this.subject = subject;
+  }
+}
```

Comment:
consider adding some getter methods (getTitle, getTimeStamp, getFirstName, etc.) to access the class variables without converting to JSON. I know you don't need the getter methods as you've currently implemented things, but it's unusual to have a class like this without them.